### ISSUE TITLE
Automate announce Techlahoma's Twitch channel going live in Slack/Twitter/FB

### ISSUE THREAD

**Issue by seejessicacode** [techlahoma/user-groups#168]
Zapier doesn't have a Twitch channel (maybe someday!!) so we'll have to use IFTTT to connect to Techlahoma's Twitch account.

@amandaharlin I already have access to Techlahoma's IFTTT account, can you lastpass the following credentials to me so I can hook this up?
- [x] Slack (info@techlahoma.org)
- [x] Twitch (info@techlahoma.org)
- [x] Buffer (Techlahoma's twitter)

**Comment by amandaharlin** [comment]
slack is hooked up. idk if i have twitch access...

**Comment by seejessicacode** [comment]
Thanks Amanda! 

@Avasond or @blazedd do either of you have info@techlahoma.org's (techlahomie) credentials you can share with me via LastPass?

**Comment by blazedd** [comment]
@Avasond has the latest password.

If you're waiting on the password to move on you can just issue a PW reset. @seejessicacode

**Comment by seejessicacode** [comment]
Thanks @blazedd! I reset info@techlahom.org's Twitch password and shared it via lastpass to you, @Avasond, and @amandaharlin.

I had a duh-doy moment and realized that `techlahomie` !== `techlahoma` and confirmed that the twitch channel is not under techlahomie's account. I tried the "forgot my username" route on Twitch with info@techlahoma.org and that's how I found `techlahomie`.

**TLDR:** Techlahoma twitch channel is not owned by info@techlahoma.org, so I can't look up username and reset password.

**Comment by blazedd** [comment]
We'll need to wait on @Avasond at this point. She setup the Twitch account.

**Comment by blazedd** [comment]
Thanks to @Avasond the account is now set to `info+twitch@techlahoma.org`. I've shared the password with you as well.

As I don't have access to `info@techlahoma.org` I cannot verify the email for the change.

@seejessicacode I sent you an invite to the Lastpass password. If there is an offical Lastpass group in techlahoma let's get it added there and re-invite everyone else back to it.

**Comment by seejessicacode** [comment]
I have verified the email and have accepted the lastpass invite. There is not an official lastpass group to my knowledge but I went ahead and shared it with @amandaharlin since she's usually the keeper of the passwords.
